content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -0,0 +1,265 @@
+---
+reviewers:
+- liggitt
+- lavalamp
+- thockin
+- smarterclayton
+title: "Deprecated API Migration Guide"
+weight: 45
+content_type: api_reference
```

Comment:
Is this content type right?

We don't have a formal list of content types (and we don't lint against that nonexistent list!) _but_ we might in future start styling API reference pages specially. How about:
```
content_type: reference
```